src/components/Layout.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Navbar from './Navbar';
+import Loader from './Loader/Loader';
+
+function Layout() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const Layout= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/ProgramKingAbel/hotel-hub-frontend/assets/57421823/04e890fe-3482-402a-92aa-687eaaf9339d)


That way, your code will meet the latest standards. 💪